ReactWindows/ReactNative.Shared/DevSupport/DevSupportManager.cs

```diff
@@ -337,13 +363,32 @@ public Task<bool> IsPackagerRunningAsync()
                 await ReloadJavaScriptFromFileAsync(progressDialog.Token);
                 dialogOperation.Cancel();
             }
+#else
+            var dialogOperation = progressDialog.ShowDialog();
+
+            if (IsRemoteDebuggingEnabled)
+            {
+                await ReloadJavaScriptInProxyMode(progressDialog.Hide, progressDialog.Token).ConfigureAwait(false);
```

Comment:
>progressDialog.Hide [](start = 50, length = 19)

Can you lift `progressDialog.Hide` and `dialogOperation.Cancel` into an variable of type Action so the bulk of this can be shared?